rxandroid/src/main/java/rx/android/app/AppObservable.java

```diff
@@ -79,13 +78,30 @@ public Boolean call(android.support.v4.app.Fragment fragment) {
         return source.observeOn(mainThread()).lift(new OperatorConditionalBinding<T, Activity>(activity, ACTIVITY_VALIDATOR));
     }
 
+    @Deprecated
+    public static <T> Observable<T> bindFragment(Object fragment, Observable<T> source) {
+        return bindFragmentInternal(fragment, source);
+    }
+
+    private static <T> Observable<T> bindFragmentInternal(Object fragment, Observable<T> source) {
```

Comment:
Destroy this method and move the respective implementations to the appropriate methods.